builtin/providers/cidr/provider_test.go

```diff
@@ -0,0 +1,48 @@
+package cidr
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/builtin/providers/aws"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+var testProviders = map[string]terraform.ResourceProvider{
+	"cidr": Provider(),
+}
+var testAccProviders map[string]terraform.ResourceProvider
+var testAccProvider *schema.Provider
+var testAccAwsProvider *schema.Provider
+
+func init() {
+	testAccProvider = Provider().(*schema.Provider)
+	testAccAwsProvider = aws.Provider().(*schema.Provider)
+	testAccProviders = map[string]terraform.ResourceProvider{
+		"cidr": testAccProvider,
+		"aws":  testAccAwsProvider,
```

Comment:
@apparentlymart 
Yeah I'm fighting the dependent provider and can't seem to make it work. There appears to be some vendor-ing naming issues, but I'll just walk away from that problem for now.
Should I refactor here to update go-cidr as well (aka update to latest and drop all those extra tests)? Or should I just make a public repo in my github and get this working then let you guys take from there?